imod_coupler/drivers/ribametamod/mapping.py

```diff
@@ -156,32 +157,56 @@ def set_metaswap_modflow_mapping(
     def set_metaswap_ribasim_mapping(
         self, packages: ChainMap[str, Any], mod2svat: Path
     ) -> None:
-        if self.coupling.rib_msw_sprinkling_map_surface_water is None:
-            return
+        table_node2svat: NDArray[np.int32]
         svat_lookup = set_svat_lookup(mod2svat)
 
         self.msw2rib = {}
 
-        table_node2svat: NDArray[np.int32] = np.loadtxt(
-            self.coupling.rib_msw_sprinkling_map_surface_water, dtype=np.int32, ndmin=2
-        )
-        rib_idx = table_node2svat[:, 0] - 1
-        msw_idx = [
-            svat_lookup[table_node2svat[ii, 1], table_node2svat[ii, 2]]
-            for ii in range(len(table_node2svat))
-        ]
-        (
-            self.msw2rib["sw_sprinkling"],
-            self.msw2rib["sw_sprinkling_mask"],
-        ) = create_mapping(
-            msw_idx,
-            rib_idx,
-            packages["msw_volume"].size,
-            packages[
-                "ribasim_nbound"
-            ],  # should become shape of 'users'-array in Ribasim
-            "sum",
-        )
+        # surface water ponding mapping
+        if self.coupling.rib_msw_ponding_map_surface_water is not None:
+            table_node2svat = np.loadtxt(
+                self.coupling.rib_msw_ponding_map_surface_water, dtype=np.int32, ndmin=2
+            )
+            rib_idx = table_node2svat[:, 0] - 1
+            msw_idx = [
+                svat_lookup[table_node2svat[ii, 1], table_node2svat[ii, 2]]
+                for ii in range(len(table_node2svat))
+            ]
+            (
+                self.msw2rib["sw_ponding"],
+                self.msw2rib["sw_ponding_mask"],
+            ) = create_mapping(
+                msw_idx,
+                rib_idx,
+                packages["msw_volume"].size,
+                packages["ribasim_nbound"],
+                "sum",
+            )
+
+        # surface water sprinkling mapping
```

Comment:
Nope, has not changed, but I changed this if...then  logic to avoid the return statement, maybe that introduced an extra indentation and as a result a reformatting by ruff. The code doing the mapping is the same as far as I can see